tldw_Server_API/tests/Audio/test_audio_transcription_retention_and_redaction.py

```
@@ -0,0 +1,346 @@
+    stt_policy = _load_stt_policy_module()
+
+    base = stt_policy.STTPolicy(
+        org_id=7,
+        delete_audio_after_success=True,
+        audio_retention_hours=0.0,
+        redact_pii=True,
+        allow_unredacted_partials=False,
+        redact_categories=["pii_email", "pii_phone"],
+    )
+
+    with pytest.raises(Exception):
```

Ruff (0.15.9)

[warning] Do not assert blind exception: `Exception`

(B017)